### ISSUE TITLE
Is this project deprecated?

### ISSUE THREAD

**Issue by woheller69** [mozilla/cookie-banner-rules-list#544]

**Comment by Croydon** [comment]
I don't think so since I recently got responses to pull requests, but it seems to have a really low priority.

Makes me wonder if Mozilla is planning to roll out this feature in Firefox Desktop anytime soon.

See also https://bugzilla.mozilla.org/show_bug.cgi?id=1783015

**Comment by Trikolon** [comment]
The future of the project is unclear at the moment unfortunately. That's also why I can't allocate much time to do reviews for the PRs. Since I don't want to make contributors wait that much I'll stop accepting changes altogether. At least until we've decided how we're going to proceed with this feature.
